openslides/utils/main.py

```diff
@@ -281,15 +281,19 @@ def start_browser(browser_url):
     Launches the default web browser at the given url and opens the
     webinterface.
     """
-    browser = webbrowser.get()
+    try:
+        browser = webbrowser.get()
+    except webbrowser.Error as e:
+        print 'Web browser controller error: %s' % e
```

Comment:
print should be called as function and not as statement. Even in python 2.6 and python 2.7